test/users-query-test.ts

```diff
@@ -0,0 +1,111 @@
+import { expect } from 'chai';
+import { USERS_QUERY } from './graphql/queries';
+import { seed } from '../src/seed/seed';
+import jwt from 'jsonwebtoken';
+import axios from 'axios';
+import prisma from '../src/prisma';
+
+describe('Users Query', function () {
+  let authToken: string;
+
+  this.beforeAll(async () => {
+    await prisma.user.deleteMany();
+    await seed();
+  });
+
+  beforeEach(async () => {
+    authToken = jwt.sign({ userID: 1 }, process.env.JWT_SECRET as string, { expiresIn: '1h' });
+  });
+
+  this.afterAll(async () => {
+    await prisma.$disconnect();
+  });
+
+  it('should return paginated users successfully', async () => {
+    const limit = 10;
+    const offset = 0;
+    const queryUsers = USERS_QUERY({ limit, offset });
+
+    const headers = { Authorization: `Bearer ${authToken}` };
+    const response = await axios.post('http://localhost:4000', queryUsers, { headers });
+
+    expect(response.data.data.Users.users.length).to.eq(10);
+    expect(response.data.data.Users.usersPreviousPage.hasMoreUsers).to.eq(false);
+    expect(response.data.data.Users.usersPreviousPage.totalUsersInPage).to.eq(0);
+    expect(response.data.data.Users.usersNextPage.hasMoreUsers).to.eq(true);
+    expect(response.data.data.Users.usersNextPage.totalUsersInPage).to.eq(41);
```

Comment:
Aqui vc tá testando a paginação mas não está testando se o array de usuários está corretos (inclusive os seus campos). Acho que tudo isso é testável com um grande `to.be.deep.eq` tendo como base o array de usuários criados pelo seed.